db/seeds.rb

```diff
@@ -1,35 +1,47 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-
 require 'open-uri'
 
-  # {
-  # location: "Arthur Avenue & East Tremont Avenue",
-  # name: "Tremont Park",
-  # type: "Bocce Courts"
-  # },
-  # => 805 parks total
-
-url = "https://data.cityofnewyork.us/resource/e4ej-j6hn.json"
+url = 'https://data.cityofnewyork.us/resource/e4ej-j6hn.json'
 json = JSON.load(open(url))
 
 json.each do |park|
-  Park.create(:name => park["name"], :location => park["location"], :activity => park["type"])  
+  park = Park.create(:name => park['name'], :location => park['location'], :facility => park['type']) unless park['type'] == 'Bathroom'
```

Comment:
Line is too long. [135/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.